File: scheduled-scala/src/main/scala/com/stulsoft/pspring/scheduleds/SomeService.scala
/*
 * Copyright (c) 2018. Yuriy Stul
 */

package com.stulsoft.pspring.scheduleds

import com.typesafe.scalalogging.LazyLogging
import org.springframework.stereotype.Component

/**
  * @author Yuriy Stul
  */
@Component
class SomeService extends LazyLogging {
  def doServiceWork(): Unit = {
    logger.info("==>doServiceWork")
    logger.info("<==doServiceWork")
  }
}

File: condition-app/src/main/java/com/stulsoft/pspring/condition/app/SomeService.java
/*
 * Copyright (c) 2020. Yuriy Stul
 */

package com.stulsoft.pspring.condition.app;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.stereotype.Service;

/**
 * @author Yuriy Stul
 */
@Service
public class SomeService {
    private static final Logger logger = LoggerFactory.getLogger(SomeService.class);

    public void test() {
        logger.info("==>test");
    }
}
